src/test/scala/cromwell/engine/backend/BackendCallSpec.scala

```diff
@@ -21,7 +21,7 @@ class BackendCallSpec extends CromwellTestkitSpec with ScalaFutures {
   "BackendCall hash function" should {
     "not change very often - if it changes, make sure it is for a good reason" in {
       val actual = backendCall.hash.futureValue.overallHash
-      val expected = "fe71298bb881a586178dca7c92fa945f"
+      val expected = "1ecd959c6081990491b214f5ae58850b"
```

Comment:
As I see it in the code all runtime attributes are included in the hash :
```scala
val orderedRuntime = Seq(
      ("docker", dockerHash getOrElse ""),
      ("defaultZones", runtimeAttributes.defaultZones.sorted.mkString(",")),
      ("failOnStderr", runtimeAttributes.failOnStderr.toString),
      ("continueOnReturnCode", runtimeAttributes.continueOnReturnCode match {
        case ContinueOnReturnCodeFlag(bool) => bool.toString
        case ContinueOnReturnCodeSet(codes) => codes.toList.sorted.mkString(",")
      }),
      ("cpu", runtimeAttributes.cpu.toString),
      ("preemptible", runtimeAttributes.preemptible.toString),
      ("defaultDisks", runtimeAttributes.defaultDisks.sortWith((l, r) => l.getName > r.getName).map(d => s"${d.getName} ${d.size} ${d.getType}").mkString(",")),
      ("memoryGB", runtimeAttributes.memoryGB.toString)
    )
```
I think we said we were going to tune this later but never got to that point yet it seems